client/blocks/site-address-changer/dialog.jsx

```diff
@@ -98,7 +146,7 @@ class SiteAddressChangerConfirmationDialog extends PureComponent {
 					/>
 					<p className="site-address-changer__confirmation-detail-copy site-address-changer__copy-deletion">
 						{ translate(
-							'{{strong}}%(currentDomainName)s{{/strong}}%(currentDomainSuffix)s will be removed and unavailable for use.',
+							'{{strong}}%(currentDomainName)s{{/strong}}%(currentDomainSuffix)s will no longer be your site address.',
```

Comment:
Worth applying "string freeze" label in the PR to ensure we have translations before deploying.

Reply:
I don't have the ability to add labels - would you mind please doing this? :)

Reply:
Someone reviewing the work can apply once we know they're final. :-)